src/tools/osdmaptool.cc

```diff
@@ -63,6 +63,8 @@ void usage()
   cout << "   --dump <format>         displays the map in plain text when <format> is 'plain', 'json' if specified format is not supported" << std::endl;
   cout << "   --tree                  displays a tree of the map" << std::endl;
   cout << "   --test-crush [--range-first <first> --range-last <last>] map pgs to acting osds" << std::endl;
+  cout << "   --adjust-crush-weight <osdid:weight>[,<osdid:weight>,<...>] change <osdid> CRUSH <weight> (but do not persist)" << std::endl;
+  cout << "   --crush-adjust-save     saves adjust-crush-weight to CRUSHmap" << std::endl;
```

Comment:
Instead of this, can we have a generate --no-update argument that skips the write, regardless of what it is?

Reply:
To make sure I follow. You're talking about removing `--crush-adjust-save` (and I guess `--upmap-save`?) and swap the behavior to update the osdmap by default unless `--no-update` is specified?

Reply:
actually.. let's remove --upmap-save and --crush-adjust-save, and add a --save.  i was thinking htere were lots of paths that set modified = true but it's just import-crush (which should continue to save) and build.

Reply:
Sounds good. I'll get to this tomorrow!